code/src/store/slices/snackbarSlice.test.tsx

```diff
@@ -0,0 +1,136 @@
+import { renderHook } from "@testing-library/react";
+
+import { useAppSelector } from "@/hooks/use-redux/useRedux";
+import snackbarReducer, {
+  _clearTimeout,
+  _setSnackbarTimerId,
+  closeSnackbar,
+  openSnackbar,
+  openSnackbarWithTimeout,
+  useIsSnackbarOpenSelector,
+  useSnackbarConfigSelector
+} from "@/store/slices/snackbarSlice";
+import { store } from "@/store/store";
+import {
+  BaseSnackbarConfig,
+  SnackbarConfigWithTimeout
+} from "@/types/snackbar.types";
+
+const mockedUseAppSelector = useAppSelector as jest.MockedFunction<
+  typeof useAppSelector
+>;
+
+jest.useFakeTimers();
+
+describe("snackbarSlice reducers", () => {
+  const initialState = {
+    isOpen: false,
+    config: {} as BaseSnackbarConfig
+  };
```

Comment:
all mocked data should move out of `describe` block

Reply:
Moved)